### ISSUE TITLE
incorrect output for interpolated classnames

### ISSUE THREAD

**Issue by mlarcher** [dlmanning/gulp-sass#410]
I am opening this issue as a reminder, as the bug doesn't reside in gulp-sass but only affects it.
The problem occurs when there are constructs like 
```
$someVar: '.className';
#{$someVar} {
    //...
  }
```
which results in
```
".className" {
    //...
}
```
Which won't be interpreted by the css parsers.
It's kind of difficult to spot at first, as it depends on your scss and can affect only a fe rules with no error output during compile.
Issue is coming from node-sass 3.4 and referenced at https://github.com/sass/node-sass/issues/1217 which in turn leads to an issue in libsass referenced at https://github.com/sass/libsass/issues/1624

Everything is fine when using node-sass 3.3.3 but the recent bump of gulp-sass to 2.1.0 forced the node-sass version to "^3.4.1", imposing this bug on all users.
I hope this helps others facing the same issue.

**Comment by xzyfer** [comment]
I am unable to reproduce this with node-sass@3.4.2. Can you confirm?

**Comment by mlarcher** [comment]
Nope, the issue is still here in node-sass 3.4.2
Tracking in libsass is now on https://github.com/sass/libsass/issues/1647

**Comment by falldowngoboone** [comment]
Apparently fixed in libsass 3.3.3? node-sass 3.4.2 is still using 3.3.2

**Comment by xzyfer** [comment]
We're still ironing out some issues. You can get LibSass 3.3.3 using the beta.
```
npm install gulp-sass@beta
```

**Comment by xzyfer** [comment]
This is fixed in v3.2.1.
